hwy/ops/x86_128-inl.h

```diff
@@ -394,6 +394,102 @@ HWY_API VFromD<D> ResizeBitCast(D d, FromV v) {
   return BitCast(d, VFromD<decltype(du8)>{detail::BitCastToInteger(v.raw)});
 }
 
+// ------------------------------ Dup128VecFromValues
+
+template <class D, HWY_IF_UI8_D(D), HWY_IF_V_SIZE_LE_D(D, 16)>
+HWY_API VFromD<D> Dup128VecFromValues(D /*d*/, TFromD<D> t0, TFromD<D> t1,
+                                      TFromD<D> t2, TFromD<D> t3, TFromD<D> t4,
+                                      TFromD<D> t5, TFromD<D> t6, TFromD<D> t7,
+                                      TFromD<D> t8, TFromD<D> t9, TFromD<D> t10,
+                                      TFromD<D> t11, TFromD<D> t12,
+                                      TFromD<D> t13, TFromD<D> t14,
+                                      TFromD<D> t15) {
+  return VFromD<D>{_mm_set_epi8(
```

Comment:
I'm curious if there is any reason not to use setr_epi8? No need to change it to that now.

Reply:
Updated x86 implementation to use _mm_setr intrinsics where available, but there are a few cases where _mm_set intrinsics needs to be used as there are no _mm_setr_epi64x, _mm512_setr_epi8, or _mm512_setr_epi16 intrinsics available on x86.

Reply:
Nice. Huh, I'm surprised that those are missing. But Highway shields users from these inconsistencies, so all good.